torch/testing/_internal/common_methods_invocations.py

```diff
@@ -16024,8 +16030,11 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
            dtypes=floating_types(),
            dtypesIfCUDA=_dispatch_dtypes(),  # histogramdd is only implemented on CPU
            sample_inputs_func=sample_inputs_histogramdd,
+           error_inputs_func=error_inputs_histogramdd,
            supports_autograd=False,
            skips=(
+               # Not implemented on CUDA
+               DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_errors', device_type='cuda'),
```

Comment:
Let's add it for the cuda path now that we're at it.

Reply:
This operator is not implemented for CUDA.

Reply:
That's a bug in the implementation of `test_errors` then I guess. That's fine.